openedx/core/djangoapps/content/block_structure/manager.py

```diff
@@ -123,7 +105,6 @@ def _update_collected(self):
                 self.modulestore,
             )
             BlockStructureTransformers.collect(block_structure)
-            self.store.add(block_structure)
```

Comment:
I'm not sure why the code in this file was modified. I'm assuming other tests broke? This seems like the code here is important to keeping the block structure correctly up to date.

Reply:
Sure we will check it.